setup/ubuntu_docker/upgrade.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+# This script upgrades dockerized Redash on Ubuntu 16.04.
+
+set -eu
+
+REDASH_BASE_PATH=/opt/redash
+REDASH_BRANCH="${REDASH_BRANCH:-master}" # Default branch/version to master if not specified in REDASH_BRANCH env var
+FILES_BASE_URL=https://raw.githubusercontent.com/getredash/redash/${REDASH_BRANCH}/setup/ubuntu_docker
+
+verify_root() {
+    # Verify running as root:
+    if [[ "$(id -u)" != "0" ]]; then
+        if [[ $# -ne 0 ]]; then
+            echo "Failed running with sudo. Exiting." 1>&2
+            exit 1
+        fi
+        echo "This script must be run as root. Trying to run with sudo."
+        sudo bash "$0" --with-sudo
+        exit 1
+    fi
+}
+
+verify_ubuntu() {
+	if [[ "$(cat /etc/lsb-release | grep "DISTRIB_ID" | awk 'BEGIN{FS="="}{print $2}')" != "Ubuntu" ]]; then
+		echo "This operating system is not Ubuntu. Exiting."
+		exit 1
+	fi
+}
+
+verify_root
+verify_ubuntu
+
+wget -O /usr/local/bin/semver $FILES_BASE_URL/files/semver
+chmod +x /usr/local/bin/semver
+
+CURRENT_IMAGE_VERSION=`docker inspect redash_server_1 | grep "Image" | grep "redash" | awk 'BEGIN{FS=":"}{print $3}' | awk 'BEGIN{FS="\""}{print $1}'`
+echo -e "Current Redash Docker image version is: $CURRENT_IMAGE_VERSION \n"
+
+REQUESTED_CHANNEL="$1"
+if [[ "$REQUESTED_CHANNEL" = "beta" ]]; then
+    AVAILABLE_IMAGE_VERSION=`curl -s "https://version.redash.io/releases?channel=beta"  | json_pp  | grep "docker_image" | head -n 1 | awk 'BEGIN{FS=":"}{print $3}' | awk 'BEGIN{FS="\""}{print $1}'`
+else
+    AVAILABLE_IMAGE_VERSION=`curl -s "https://version.redash.io/api/releases?channel=stable"  | json_pp  | grep "docker_image" | head -n 1 | awk 'BEGIN{FS=":"}{print $3}' | awk 'BEGIN{FS="\""}{print $1}'`
+fi
+
+echo -e "Available Redash Docker image version is: $AVAILABLE_IMAGE_VERSION \n"
+
+var=`semver compare $CURRENT_IMAGE_VERSION $AVAILABLE_IMAGE_VERSION`
+if [[ "$var" = "-1" ]]; then
+    echo "There is a newer version of Redash Docker Image"
+    read -p "Do you want to upgrade it?  [Y/n] : " doUpgrade
+    if [[ "$doUpgrade" = "y" || "$doUpgrade" = "Y" ]]; then
+        if [[ -e $REDASH_BASE_PATH/.env ]]; then
+            count=`cat $REDASH_BASE_PATH/.env | grep "TAG=" | wc -l`
+            if [[ $count -gt 0 ]]; then
+                sed -i "/TAG=*/c TAG=${AVAILABLE_IMAGE_VERSION}" $REDASH_BASE_PATH/.env
+            else
+                echo "TAG=${AVAILABLE_IMAGE_VERSION}" >> $REDASH_BASE_PATH/.env
+            fi
+        else
+            touch $REDASH_BASE_PATH/.env
+            echo "TAG=${AVAILABLE_IMAGE_VERSION}" >> $REDASH_BASE_PATH/.env
+        fi
+        cd $REDASH_BASE_PATH
+        docker stop redash_server_1 redash_worker_1 > /dev/null
+        docker rm redash_server_1 redash_worker_1 > /dev/null
```

Comment:
we stop particular container,  that is why use docker command

Reply:
```
 docker-compose stop --help
Stop running containers without removing them.

They can be started again with `docker-compose start`.

Usage: stop [options] [SERVICE...]

Options:
  -t, --timeout TIMEOUT      Specify a shutdown timeout in seconds.
                             (default: 10)
```

docker-compose has a stop command as well, and it's better to use it as you don't have to guess how it will name the containers.